drivers/clock_control/clock_control_tt_bh.c

```diff
@@ -344,13 +344,13 @@ static void clock_control_tt_bh_update(const struct clock_control_tt_bh_config *
 	/* Setup external postdivs */
 	clock_control_tt_bh_config_ext_postdivs(config, settings);
 
-	k_busy_wait(300);
+	k_busy_wait_ns(300);
```

Comment:
See my comment about the implementation of `k_busy_wait_ns`. Since this effectively works out to `k_busy_wait(1)`, should we just use that here? Or is my timer math bad here?

Reply:
It rounds up to 1 us in the half-assed implementation which was my suggestion, but it will work for the time being and be API compatible with a proper solution that eventually gets upstreamed.